### PR TITLE
Update @plone/* packages info

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: PACKAGES.md [hunk 1]
```diff
@@ -2,17 +2,37 @@
...
+Most of them are experimental and are marked in their respective `README` files.
+Plone 6.1 (Volto 18) depends on:
+- `@plone/registry`
+- `@plone/scripts`
+- `@plone/volto-slate`
+
+and as a development dependency:
+- `@plone/types`
+
+Plone 6 (Volto 17 and below) does not use any of them.
```
Comment (stevepiercy) [inline PACKAGES.md:15]:
```suggestion
Plone 6.0.x (Volto 17 and below) does not use any of them.
```

File: PACKAGES.md [hunk 2]
```diff
@@ -2,17 +2,37 @@
 
 This document describes the packages that come with Volto, the default frontend for Plone 6.
 
+These packages are part of Plone's API-first story.
+Most of them are experimental and are marked in their respective `README` files.
+Plone 6.1 (Volto 18) depends on:
```
Comment (stevepiercy) [inline PACKAGES.md:7]:
```suggestion
Plone 6.1.x (Volto 18) depends on:
```

File: PACKAGES.md [hunk 3]
```diff
@@ -2,17 +2,37 @@
...
+- `@plone/registry`
+- `@plone/scripts`
+- `@plone/volto-slate`
+
+and as a development dependency:
+- `@plone/types`
+
+Plone 6 (Volto 17 and below) does not use any of them.
+
+These packages are expected to be used and part of Plone 7.
```
Comment (stevepiercy) [inline PACKAGES.md:17]:
```suggestion
These packages are expected to be used and become part of Plone 7.
```

File: PACKAGES.md [hunk 4]
```diff
@@ -2,17 +2,37 @@
...
+Could be that some of them will be part of the upcoming Plone minor versions prior to Plone 7.
```
Comment (stevepiercy) [inline PACKAGES.md:18]:
```suggestion
Some of them might become part of Plone 6.1.x minor versions.
```

File: PACKAGES.md [hunk 5]
```diff
@@ -2,17 +2,37 @@
...
+
+The packages are divided in three categories or types:
```
Comment (stevepiercy) [inline PACKAGES.md:20]:
```suggestion
The packages are divided into three categories or types:
```

File: PACKAGES.md [hunk 6]
```diff
@@ -29,11 +49,11 @@ They must be published and bundled in a traditional (transpiled) way.
...
 
 
-## Utility packages
+## Utilities packages
 
--   `@plone/drivers`
--   `@plone/helpers`
 -   `@plone/providers`
+-   `@plone/helpers`
+-   `@plone/drivers`
```
Comment (stevepiercy) [inline PACKAGES.md:56]: Can you leave these sorted alphabetically?

File: PACKAGES.md [hunk 7]
```diff
@@ -44,19 +64,22 @@ They must be published in the traditional way, as a bundle.
...
 
 
-## Feature packages
+## Add-on packages
 
 -   `@plone/blocks`
--   `@plone/contents`
 -   `@plone/slots`
+-   `@plone/theming`
+-   `@plone/contents`
```
Comment (stevepiercy) [inline PACKAGES.md:72]: Can you leave these sorted alphabetically?

File: PACKAGES.md [hunk 8]
```diff
@@ -44,19 +64,22 @@ They must be published in the traditional way, as a bundle.
...
 
 
 ### Rules
 
-Feature packages, or add-on packages, can depend on any other package.
+Add-on or feature packages, can depend on any other package.
 You must distribute them as source code, and not transpile them.
 They must provide a default configuration registry loader as the default main entry point export.
-They must be loadable as any other add-on.
+Unlike Volto add-ons, it is preferibly *NOT* to place the code in `src` folder.
```
Comment (stevepiercy) [inline PACKAGES.md:80]:
```suggestion
Unlike Volto add-ons, do *NOT* place the code in the `src` folder.
```

File: PACKAGES.md [hunk 9]
```diff
@@ -44,19 +64,22 @@ They must be published in the traditional way, as a bundle.
...
+If the package is not being transpiled, the direct resolution must work OOTB, so the bundlers resolution is direct, as well as the TypeScript resolution.
```
Comment (stevepiercy) [inline PACKAGES.md:81]:
```suggestion
If you do not transpile the package, the direct resolution must work out of the box, where both the bundler and TypeScript resolution are direct.
```

File: PACKAGES.md [hunk 10]
```diff
@@ -44,19 +64,22 @@ They must be published in the traditional way, as a bundle.
...
+They must be loadable as any other add-on and contain an add-on registry compatible `install`-able default export.
```
Comment (stevepiercy) [inline PACKAGES.md:82]:
```suggestion
They must be loadable as any other add-on, and contain an add-on registry compatible `install`-able default export.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Please double-check my suggestions for accuracy.

File: PACKAGES.md (hunk 6 above)
Comment (sneridagh) [reply PACKAGES.md:56]: I've ordered by relevance to me, due to the current state of things (drivers and rsc are not a thing yet). It just occurred to me that if they do not exist yet, let's remove them.

File: PACKAGES.md (hunk 7 above)
Comment (sneridagh) [reply PACKAGES.md:72]: Same as above, but if you think it's valuable, let's do it.

Comment (stevepiercy) [reply PACKAGES.md:72]: Yeah, relevance is arbitrary to the developer. Alpha also makes it easier to find in a list.

**Review by stevepiercy — APPROVED** [review]
After resolving sorting of thingies, I approve, too.

**Pull request merged**